### ISSUE TITLE
Benchmark compile times

### ISSUE THREAD

**Issue by Ralith** [TomGillen/ecs_bench_suite#6]
Given rust's reputation in this area, it would be useful to publish how long both clean (fresh checkout) and incremental (`touch` every .rs file but don't change anything) builds of the suite take. This could be accomplished by feature-guarding each lib, and for maximum clarity in the results, omitting tests that pull in heavy deps like serde which might otherwise be avoided.

**Comment by kettle11** [comment]
I strongly agree! It'd be great if compile times were included in most Rust benchmarks. It's an important factor to weigh when evaluating a Rust library.
